test/src/stakeholder/actions/reportDoubleVote.ts

```diff
@@ -46,16 +46,16 @@ export default class ReportDoubleVote {
         return "reportDoubleVote";
     }
 
-    public criminal(): H512 {
-        const { message1 } = this;
-        const digest = blake256(RLP.encode(messageToEncodeObject(message1)[0]));
-        const signature = message1.signature.toString();
-        const pubkey = recoverSchnorr(digest, {
-            r: signature.slice(0, 64),
-            s: signature.slice(64)
-        });
-        return H512.ensure(pubkey);
-    }
+    // public criminal(): H512 {
+    //     const { message1 } = this;
+    //     const digest = blake256(RLP.encode(messageToEncodeObject(message1)[0]));
+    //     const signature = message1.signature.toString();
+    //     const pubkey = recoverSchnorr(digest, {
+    //         r: signature.slice(0, 64),
+    //         s: signature.slice(64)
+    //     });
+    //     return H512.ensure(pubkey);
+    // }
```

Comment:
If this code is not used, let's remove it.

Reply:
Thank you, I removed it. I'll force-push after your review ends

Reply:
I reviewed it. Other code looks good to me.

Reply:
Ok, I applied the change